tests/ha/fencing.pm

```diff
@@ -32,8 +32,8 @@ sub run {
         # Fence the node
         assert_script_run 'crm -F node fence ' . get_node_to_join;
 
-        # Wait a little to be sure that fence command is on his way
-        sleep 60;
+        # We also need to reset the console on this node
```

Comment:
Why?

Reply:
For the sleep? Is not needed in fact, I did multiple test on x86_64 and aarch64 to be sure, so removing an uneeded sleep is beeter, no?

Reply:
Yeah, I guess @okurz cannot express himself friendly and clear sometimes, but I guess he is referring to your comment. To be honest for me it's not clear from the comment why we need to reset consoles on this node, obviously, because I don't have enough experience with HA, but basically having reset_consoles already means that we need to reset the consoles on this node. Do we have a chance to explain the reasoning a little bit better?

Reply:
Comment it's because on all other nodes but 2nd node I do a reset_consoles and on 2nd node I fence the first node and after I want to have the 2nd node like the others, so I also do a `reset_consoles`.

Reply:
It would be great if you could put exactly this explanation as a comment, or even remove it completely, as now it's consistent. Cheers!

Reply:
Yes, new code is under test!